specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -1773,6 +1829,104 @@
         }
       }
     },
+    "AcsCallParticipantBaseEventData": {
+      "description": "Schema of common properties of participant events",
+      "type": "object",
+      "properties": {
+        "user": {
+          "description": "The communication identifier of the participant",
+          "$ref": "../../../../../communication/data-plane/Common/stable/2023-11-15/common.json#/definitions/CommunicationIdentifierModel"
+        },
+        "displayName": {
+          "description": "The display name of the participant.",
+          "type": "string"
+        },
+        "participantId": {
+          "description": "The Id of the participant.",
+          "type": "string"
+        },
+        "userAgent": {
+          "description": "The user agent of the participant. ",
+          "type": "string"
+        }
+      }
+    },
+    "AcsCallingEventBaseProperties": {
+      "description": "Schema of common properties of all calling events",
+      "type": "object",
+      "properties": {
+        "startedBy": {
```

Comment:
this is also should be call participant model?
![{01254C29-AC9F-4838-990A-D5B17EEEAFE7}](https://github.com/user-attachments/assets/80df0efc-a613-4481-82ac-00003dcad521)